lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -91,6 +91,10 @@ const char *SwiftLanguageRuntime::GetStandardLibraryBaseName() {
   return "swiftCore";
 }
 
+const char *SwiftLanguageRuntime::GetConcurrencyLibraryBaseName() {
```

Comment:
nit: Can this return a `StringRef` or `StringLiteral` ?

Reply:
It could, for sure, but it would break with the pattern of the other functions above. We can do a cleanup and change all of them in a separate PR